style.css

```diff
@@ -601,11 +601,13 @@ input:focus:invalid {
  #container1 {
     height: 100vh;
     background-color: #1c1a19;
-    background-image: url("images/Illustration-header.png");
+    background-image: url(images/Illustration-header.png);
     background-repeat: no-repeat;
     background-position: right;
+    background-position-y: 0.2rem;
     background-size: 40%;
     padding-bottom: 3rem;
+       
```

Comment:
- [ ] This is not the portion I am making the comments on. But because, those areas are cannot be highlighted. I will leave all the comments here with screenshots that will direct you where to make changes on your projects.

- [REQUIRED] It is required to stick to the figma design as close as possible. This includes margin, padding, typography, font among others. Below are the screenshots of figma designs and your designs on every section where changes are required. Kindly compare the two screenshots on each section and walk your designs alongside the figma designs

###  Your Design (Recent Works)
![your-project-works](https://github.com/Tshepo45/Portfolio-desktop-version/assets/26597432/f053e466-625e-40e1-9d2f-a408664398a9)

### Figma Design 
![figma-project-works](https://github.com/Tshepo45/Portfolio-desktop-version/assets/26597432/bf326814-559b-4f2b-a3d5-6800ed7520f0)


### Your Design (Project Lists)
![your-project-lists](https://github.com/Tshepo45/Portfolio-desktop-version/assets/26597432/4212a051-2906-42a6-a219-32ebf2771649)

### Figma Design
![figma-project-lists](https://github.com/Tshepo45/Portfolio-desktop-version/assets/26597432/2b65d72f-f054-43f6-85ce-9075cf3d3484)


### Your Design (Lang, Framework, Skills)
![your-lang-lists](https://github.com/Tshepo45/Portfolio-desktop-version/assets/26597432/5cef21f7-49a9-48bd-b5fd-9a91b1162a9f)

### Figma Design
![figma-lang-lists](https://github.com/Tshepo45/Portfolio-desktop-version/assets/26597432/39748428-fcab-4821-8e44-a5d6b6ff0984)


### Your Design (Form)
![your-form](https://github.com/Tshepo45/Portfolio-desktop-version/assets/26597432/cecc65f4-5ed6-4949-a800-605d0071ef83)

### Figma Design
![figma-form](https://github.com/Tshepo45/Portfolio-desktop-version/assets/26597432/62321cd9-f185-4f8a-8ab5-bb78dbec8782)